texthero/preprocessing.py

```diff
@@ -680,12 +708,12 @@ def tokenize(s: pd.Series) -> pd.Series:
 
     """
 
-    punct = string.punctuation.replace("_", "")
     # In regex, the metacharacter 'w' is "a-z, A-Z, 0-9, including the _ (underscore) character." We therefore remove it from the punctuation string as this is already included in \w
+    punct = string.punctuation.replace("_", "")
 
-    pattern = rf"((\w)([{punct}])(?:\B|$)|(?:^|\B)([{punct}])(\w))"
-
-    return s.str.replace(pattern, r"\2 \3 \4 \5").str.split()
+    return s.str.replace(
+        _get_pattern_for_tokenisation(punct), r"\2 \3 \4 \5"
```

Comment:
As far as I can see, you could immediately create the pattern for tokenization above where `_get_pattern_for_tokenisation_` is defined with

```
punct = string.punctuation.replace("_", "")
TOKENIZATION = rf"((\w)([{punct}])(?:\B|$)|(?:^|\B)([{punct}])(\w))"
```

so we don't need an extra function to get this pattern and everything looks nice.